web-local/src/lib/components/workspace/explore/metrics-container/MeasuresContainer.svelte

```diff
@@ -0,0 +1,139 @@
+<script lang="ts">
+  import {
+    useRuntimeServiceMetricsViewTotals,
+    V1MetricsViewTotalsResponse,
+  } from "@rilldata/web-common/runtime-client";
+  import { runtimeStore } from "@rilldata/web-local/lib/application-state-stores/application-store";
+  import { useMetaQuery } from "@rilldata/web-local/lib/svelte-query/dashboards";
+  import { EntityStatus } from "@rilldata/web-common/lib/entity";
+  import type { UseQueryStoreResult } from "@sveltestack/svelte-query";
+  import {
+    MetricsExplorerEntity,
+    metricsExplorerStore,
+  } from "../../../../application-state-stores/explorer-stores";
+  import MeasureBigNumber from "./MeasureBigNumber.svelte";
+
+  export let metricViewName;
+  export let metricsContainerHeight;
```

Comment:
why does this need to be done as a prop here rather htan in the `MeasuresContainer.svelte` component itself?

Reply:
I am unable to recall why I did this exactly but I think adding a full-sized parent to `MeasuresContainer.svelte` should work. Making the change.